backend/corpora/common/corpora_orm.py

```diff
@@ -122,11 +122,13 @@ class ProjectLinkType(enum.Enum):
 
     PROTOCOL - A link to a sequencing protocol.
     RAW_DATA - A link to a raw data repository.
+    SUMMARY - The "summary" link cellxgene should display.
     OTHER - Other.
     """
 
     PROTOCOL = "Protocol"
     RAW_DATA = "Raw data"
+    SUMMARY = "Summary"
```

Comment:
Enums are handles at in the python code, not in the DB schema, so migration is not required.

Reply:
So after poking around a bit, I think the Enums are also enforced by postgres as a column type. I added a migration script.

Reply:
Uhh those these string do not match...so now I'm not sure what's going on.

Reply:
Oh nvm, we pass `name` not `value`.